packages/core-data/src/hooks/use-entity-record.ts

```diff
@@ -23,13 +23,18 @@ interface EntityRecordResolution< RecordType > {
 	status: Status;
 }
 
+interface Options {
+	__experimentalEnabled: boolean;
```

Comment:
Why do we want the option to be experimental? The hook itself is experimental, I don't think we have to experiment with it twice (inception noises) 😆 ?